src/cfnlint/data/ExtendedSpecs/us-east-1/05_pricing_property_values.json

```diff
@@ -455,13 +456,404 @@
+      "cache.m3.medium",
+      "cache.m3.xlarge",
+      "cache.m4.10xlarge",
+      "cache.m4.2xlarge",
+      "cache.m4.4xlarge",
+      "cache.m4.large",
+      "cache.m4.xlarge",
+      "cache.m5.12xlarge",
+      "cache.m5.24xlarge",
+      "cache.m5.2xlarge",
+      "cache.m5.4xlarge",
+      "cache.m5.large",
+      "cache.m5.xlarge",
+      "cache.r3.2xlarge",
+      "cache.r3.4xlarge",
+      "cache.r3.8xlarge",
+      "cache.r3.large",
+      "cache.r3.xlarge",
+      "cache.r4.16xlarge",
+      "cache.r4.2xlarge",
+      "cache.r4.4xlarge",
+      "cache.r4.8xlarge",
+      "cache.r4.large",
+      "cache.r4.xlarge",
+      "cache.r5.12xlarge",
+      "cache.r5.24xlarge",
+      "cache.r5.2xlarge",
+      "cache.r5.4xlarge",
+      "cache.r5.large",
+      "cache.r5.xlarge",
+      "cache.t1.micro",
+      "cache.t2.medium",
+      "cache.t2.micro",
+      "cache.t2.small",
+      "cache.t3.medium",
+      "cache.t3.micro",
+      "cache.t3.small"
+    ]
+  },
+  {
+    "op": "add",
+    "path": "/ValueTypes/ElasticsearchInstanceType/AllowedValues",
+    "value": [
+      "c4.2xlarge.elasticsearch",
+      "c4.4xlarge.elasticsearch",
+      "c4.8xlarge.elasticsearch",
+      "c4.large.elasticsearch",
+      "c4.xlarge.elasticsearch",
+      "c5.18xlarge.elasticsearch",
+      "c5.2xlarge.elasticsearch",
+      "c5.4xlarge.elasticsearch",
+      "c5.9xlarge.elasticsearch",
+      "c5.large.elasticsearch",
+      "c5.xlarge.elasticsearch",
+      "i2.2xlarge.elasticsearch",
+      "i2.xlarge.elasticsearch",
+      "i3.16xlarge.elasticsearch",
+      "i3.2xlarge.elasticsearch",
+      "i3.4xlarge.elasticsearch",
+      "i3.8xlarge.elasticsearch",
+      "i3.large.elasticsearch",
+      "i3.xlarge.elasticsearch",
+      "m3.2xlarge.elasticsearch",
+      "m3.large.elasticsearch",
+      "m3.medium.elasticsearch",
+      "m3.xlarge.elasticsearch",
+      "m4.10xlarge.elasticsearch",
+      "m4.2xlarge.elasticsearch",
+      "m4.4xlarge.elasticsearch",
+      "m4.large.elasticsearch",
+      "m4.xlarge.elasticsearch",
+      "m5.12xlarge.elasticsearch",
+      "m5.2xlarge.elasticsearch",
+      "m5.4xlarge.elasticsearch",
+      "m5.large.elasticsearch",
+      "m5.xlarge.elasticsearch",
+      "r3.2xlarge.elasticsearch",
+      "r3.4xlarge.elasticsearch",
+      "r3.8xlarge.elasticsearch",
+      "r3.large.elasticsearch",
+      "r3.xlarge.elasticsearch",
+      "r4.16xlarge.elasticsearch",
+      "r4.2xlarge.elasticsearch",
+      "r4.4xlarge.elasticsearch",
+      "r4.8xlarge.elasticsearch",
+      "r4.large.elasticsearch",
+      "r4.xlarge.elasticsearch",
+      "r5.12xlarge.elasticsearch",
+      "r5.2xlarge.elasticsearch",
+      "r5.4xlarge.elasticsearch",
+      "r5.large.elasticsearch",
+      "r5.xlarge.elasticsearch",
+      "t2.medium.elasticsearch",
+      "t2.micro.elasticsearch",
+      "t2.small.elasticsearch",
+      "ultrawarm1.large.elasticsearch",
+      "ultrawarm1.medium.elasticsearch"
+    ]
+  },
+  {
```

Comment:
https://github.com/aws-cloudformation/cfn-python-lint/pull/1541 seems to be working based on this file 🎉